Окуньков Семен

WisdomSpirit opened the pull request:
[Посмотреть решение](https://hruview.herokuapp.com/WisdomSpirit/markup-task-5/master/index.html)

FoOkySNick committed:
Add files via upload

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

dimastark commented on index.css:
```diff
@@ -0,0 +1,287 @@
+.article-title
+{
+    padding: 10px;
```
Давай попробуем использовать csscomb? А то как-то слишком много стилей, сложно читать.

dimastark commented on index.html:
```diff
@@ -1,9 +1,357 @@
     <title>Задача «Мурмаркет»</title>
 </head>
 <body>
+    <button class="list-button"></button>
```
А зачем эта кнопка?

WisdomSpirit replied:
забыл убрать с прошлого задания, где котики по нажатию кнопки в список превращались

dimastark commented on index.html:
```diff
@@ -1,9 +1,357 @@
 </head>
 <body>
+    <button class="list-button"></button>
+    <header class="article-title"><a>К</a>отики <a>Б</a>aбы <a>К</a>лавы</header>
```
Может лучше спаном?

dimastark reviewed (commented):
Не работает
* sticky-footer. Сейчас он работает также как шапка.
* Ссылки в футере не все имеют правильный цвет, как в условии.
![image](https://user-images.githubusercontent.com/11780431/48088559-2a3ad980-e224-11e8-856b-ee7ffac763ae.png)
https://validator.w3.org/nu/?doc=https%3A%2F%2Fhruview.herokuapp.com%2FWisdomSpirit%2Fma
🍅

FoOkySNick committed:
Add files via upload

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

dimastark commented on index.html:
```diff
@@ -1,9 +1,372 @@
+                <a class="koto-name" href="#win1">Persival</a>
+            </section>
+            <a class="arrow arrow-prev" href="#target-item-10">
+                <img src="cat-10.png" width="320" height="200" alt="">
```
Надо title и alt.

dimastark reviewed (commented):
Давай все таки сделаем хотя бы для 1000px экрана.
* Модальное окно не похоже на то, что на макете
* Модальное окно не оцентрировано по вертикали

dimastark commented:
Футер работает неправильно.

FoOkySNick committed:
очень старался, вроде как все исправил:)

теперь при добавлении большого лорема ипсума футер едет вниз за контентом, центрировал по вертикали мадальное окно, сделал его похожим на макет

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

dimastark commented:
* Модалка открывается только при клике на имя
* Мы еще вроде хотели сделать хотя бы для 1000px экрана (пока не работает)

В остальном все супер! Очень круто, что сделал закрытие модалки не только на крестик, но и при клике на паранджу.

FoOkySNick committed:
футер в модалке стал работать адекватнее

модалка и так у меня вроде как открывается только по нажатию на имя котика, а про 1000px - не понял что имелось в виду

honest-hrundel commented:
:tomato: Не пройден линтинг или базовые тесты

FoOkySNick committed:
убрал тестовый лорем ипсум, чтобы линтер не ругался

honest-hrundel commented:
:green_apple: Пройден линтинг и базовые тесты

dimastark commented:
🍏
